src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.netcoreapp.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the http://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class NativeLibrary
+    {
+        public static void Free(System.IntPtr handle) { }
+        public static System.IntPtr GetExport(System.IntPtr handle, string name) { throw null; }
+        public static System.IntPtr Load(string libraryPath) { throw null; }
+        public static System.IntPtr Load(string libraryName, System.Reflection.Assembly assembly, DllImportSearchPath? searchPath) { throw null; }
+        public static bool TryGetExport(IntPtr handle, string name, out IntPtr address) { throw null; }
+        public static bool TryLoad(string libraryPath, out System.IntPtr handle) { throw null; }
+        public static bool TryLoad(string libraryName, System.Reflection.Assembly assembly, DllImportSearchPath? searchPath, out System.IntPtr handle) { throw null; }
+    }
+}
```

Comment:
Are there tests to go along with this?

Reply:
Tests are already in CoreCLR repo: 
https://github.com/dotnet/coreclr/blob/master/tests/src/Interop/NativeLibrary/NativeLibraryTests.cs

Reply:
Does Mono reuse CoreCLR tests, like they do CoreFX tests? @marek-safar

Reply:
> Does Mono reuse CoreCLR tests

Yes, Mono runs snapshot of CoreCLR tests. BTW: https://github.com/mono/mono/issues/9640 tracks updating that snapshot

In general, the large part of interop test coverage is in CoreCLR. CoreFX does not have the specific infrastructure required to building and run interop tests.

Reply:
There are few places in CoreFX, where these APIs could be used:
https://github.com/dotnet/corefx/search?l=csharp&q=dlopen

Reply:
In particular, it'd be good to use System.Drawing as validation for this.